journal-fur-medienlinguistik.csl

```diff
@@ -0,0 +1,204 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Journal für Medienlinguistik (Deutsch)</title>
+    <title-short>jfml (Deutsch)</title-short>
+    <id>http://www.zotero.org/styles/journal-fur-medienlinguistik</id>
+    <link href="http://www.zotero.org/styles/journal-fur-medienlinguistik" rel="self"/>
+    <link href="http://www.zotero.org/styles/technische-universitat-dresden-linguistik" rel="template"/>
+    <link xml:lang="de" href="https://jfml.org/libraryFiles/downloadPublic/5" rel="documentation"/>
+    <author>
+      <name>Simon Meier-Vieracker</name>
+      <uri>https://tu-dresden.de/gsw/slk/germanistik/al</uri>
+      <email>simon.meier-vieracker@tu-dresden.de</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="linguistics"/>
+    <issn>2569-6491</issn>
+    <summary>Stylesheet des Journals für Medienlinguistik (jfml)</summary>
+    <updated>2022-06-22T06:36:28+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de-DE">
+    <terms>
+      <term name="et-al">et al.</term>
+      <term name="editor" form="short">Hg.</term>
+      <term name="online">Online unter</term>
+    </terms>
+  </locale>
+  <macro name="creator-year-long">
+    <choose>
+      <if match="none" variable="author editor">
+        <text value="o. A." suffix=" "/>
+      </if>
+    </choose>
+    <group delimiter=" ">
+      <names variable="author">
+        <name delimiter="/" name-as-sort-order="all"/>
+        <label form="short" prefix=" (" suffix=")"/>
+        <substitute>
+          <names variable="editor"/>
+        </substitute>
+      </names>
+      <date date-parts="year" form="text" variable="issued" prefix="(" suffix=")"/>
+    </group>
+  </macro>
+  <macro name="editor-container-title">
+    <group delimiter=": " suffix=" ">
+      <text term="in" text-case="capitalize-first"/>
+      <group delimiter=": " suffix=".">
+        <names variable="editor">
+          <name delimiter="/" name-as-sort-order="all"/>
+          <label form="short" prefix=" (" suffix=")"/>
+        </names>
+        <text macro="container-title"/>
+      </group>
+    </group>
+  </macro>
+  <macro name="publication-place-publisher">
+    <group delimiter=" ">
+      <text variable="publisher-place" suffix=":"/>
+      <text variable="publisher" quotes="false" font-variant="normal" font-weight="normal" vertical-align="baseline"/>
+      <text macro="collection-title-number"/>
+    </group>
```

Comment:
```suggestion
    <group delimiter=" ">
      <group delimiter=": ">
        <text variable="publisher-place" suffix=":"/>
        <text variable="publisher"/>
      </group>
      <text macro="collection-title-number"/>
    </group>
```

Always more robust to use punctuation in delimiters (here, in case there's no publisher for some reason)